src/doom/d_main.c

```diff
@@ -980,7 +980,7 @@ void PrintDehackedBanners(void)
     }
 }
 
-static struct 
+static struct
```

Comment:
Please keep these unrelated whitespace changes out of the patch.

Reply:
I've forced push the branch after removing the separate commit with the whitespace changes. These changes can still be found in this commit as of now: https://github.com/NY00123/chocolate-doom/commit/e81ea320d47586ef7f0f945f6356950b95cd5dc8

I originally stumbled upon these in the Hexen tree after copying similar code from Doom or Strife. Since it's preferred to not add new trailing whitespace, and I copied the ones from Doom/Strife, I had to remove them in Hexen, so I thought that I could adjust Doom and Strife for consistency.